src/main/java/teammates/common/datatransfer/attributes/AccountAttributes.java

```diff
@@ -101,19 +115,20 @@ public void setCreatedAt(Instant createdAt) {
 
         addNonEmptyError(FieldValidator.getInvalidityInfoForEmail(email), errors);
 
-        // No validation necessary for createdAt field.
+        // No validation necessary for createdAt and readNotifications fields.
 
         return errors;
     }
 
     @Override
     public Account toEntity() {
-        return new Account(googleId, name, email);
+        return new Account(googleId, name, email, readNotifications);
     }
 
     @Override
     public String toString() {
-        return JsonUtils.toJson(this, AccountAttributes.class);
+        return "AccountAttributes [googleId=" + googleId + ", name=" + name
```

Comment:
Why is the `toJson` method removed? In any case, the string now is not a JSON representation of the object

Reply:
This was done both for consistency with most of the other attribute classes, and to exclude the `readNotifications` page in the string representation of `AccountAttributes`.